### ISSUE TITLE
Not able to find custom folder in my .jupyter dir

### ISSUE THREAD

**Issue by jaspindersingh83** [jupyter/notebook#2296]
Hi Team 

I am trying to change autosave settings in my jupyter/custom/custom.js directory but there is no custom directory in my jupyter

**Comment by takluyver** [comment]
The jupyter folder in question should be `~/.jupyter` (in your home directory, with a leading `.`). if the `custom` directory doesn't exist inside that, just create it.

**Comment by LoveJS** [comment]
I am trying to find ~/.jupyter/jupyter_notebook_config.py.  
I am using Linux box, and when I went to Home folder, there is nothing that starts with .jupyter.  Is it somewhere hidden? 
I typed   find . -name "jupyter_notebook_config.py"  and it generated following:
./.jupyter/jupyter_notebook_config.py

**Comment by madhavij** [comment]
@takluyver : That is the case when ~/.jupyter is already present. In my case the parent .jupyter dir is not present. Is it a good practice to manually create this directory? (I'm trying to customise the notebook profile )

**Comment by takluyver** [comment]
Yup, feel free to create it. You can also run `jupyter notebook --generate-config` to write out the config files with all the options commented out.

**Comment by RRosio** [comment]
Hello, I'm closing this issue as it has had no updates in the last 365 days. Please feel free to reopen this issue if it still needs to be addressed. Thank you!
